playground/frontend/lib/modules/editor/components/share_dropdown/share_dropdown_body.dart

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:playground/modules/editor/components/share_dropdown/share_tabs/share_tabs.dart';
+import 'package:playground/modules/editor/components/share_dropdown/share_tabs_headers.dart';
+import 'package:playground/modules/output/components/output_header/tab_header.dart';
+
+const _kTabsCount = 2;
+
+class ShareDropdownBody extends StatefulWidget {
+  const ShareDropdownBody({super.key});
+
+  @override
+  State<ShareDropdownBody> createState() => _ShareDropdownBodyState();
+}
+
+class _ShareDropdownBodyState extends State<ShareDropdownBody>
+    with SingleTickerProviderStateMixin {
+  late final tabController = TabController(vsync: this, length: _kTabsCount);
+
+  @override
+  void dispose() {
+    tabController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        TabHeader(
+          tabController: tabController,
+          tabsWidget: ShareTabsHeaders(tabController: tabController),
+        ),
+        Expanded(
+          child: ShareTabs(tabController: tabController),
```

Comment:
@pabloem / @alexeyinkin I think using the package seems reasonable enough but for custom packages should we have them hosted under https://github.com/akvelon instead?

Reply:
@pabloem  / @alexeyinkin  I think using the package seems reasonable enough but for custom packages should we have them hosted under https://github.com/akvelon instead?

Reply:
I made it before starting the work at Akvelon, and it gained 58% popularity before that. We can technically fork it to the Akvelon repo, but then it will split the user base with no clear benefit.

Also we do use other packages by 3rd parties:
https://pub.dev/packages/aligned_dialog
https://pub.dev/packages/expansion_widget
https://pub.dev/packages/highlight

and do not attempt to own them although the last one is nearly abandoned (there's just no alternative).
Although we are forking https://pub.dev/packages/code_text_field to take ownership of the new text editor features.

I would prefer not using `keyed_collection_widgets` at all than forking, as the fork would weaken maintainability.

Reply:
the package lgtm being that it's mit-licensed and so on

Reply:
Then I suggest you merge it as it is when the review is otherwise done, because it is hard to track so many changes.
I filed a follow-up issue for these tabs: https://github.com/apache/beam/issues/22663

Reply:
Responding to @damondouglas 

> @pabloem / @alexeyinkin I think using the package seems reasonable enough but for custom packages should we have them hosted under https://github.com/akvelon instead?

I don't know what are the options. This discussion doesn't need to block the PR. We can discuss this next meeting?

Reply:
I saw this:  https://dart.dev/tools/pub/publishing#publishing-is-forever
So I'm fine.  Thank you for listening.